CMakeLists.txt

```diff
@@ -416,6 +416,15 @@ endif()
 # Boost (required)
 include("${PCL_SOURCE_DIR}/cmake/pcl_find_boost.cmake")
 
+# System zlib (for nurbs on surface)
+option(WITH_SYSTEM_ZLIB "Use system zlib" TRUE)
+if(WITH_SYSTEM_ZLIB)
+  find_package(ZLIB)
```

Comment:
Add **required** here - and remove the find_package call in surface/CMakeLists.txt.

Reply:
This would imply that users must have zlib development packages (okay, common) or change the option.
My implementation was chosen so that the absence of zlib doesn't break configuration until the single component which uses zlib is actually enabled.
But I can change it.

Reply:
ahh true. Lets keept it as is then 👍